nilearn/signal.py

```diff
@@ -417,6 +417,13 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
         nilearn.image.clean_img
     """
 
+    if isinstance(low_pass, bool):
+        raise TypeError("low pass must be float or None but you provided "
+                        "low_pass='{0}'".format(low_pass))
```

Comment:
string is expressed in quotes actually.

We will wait for final reviews from @lesteve @AlexandreAbraham